### PR TITLE
2020 Contributor Survey

### PR DESCRIPTION
**Pull request by mark-i-m**
**NOTE TO SELF**: actually open the survey form to responses.

r? @nikomatsakis

### CODE REVIEW

File: posts/inside-rust/2020-05-27-contributor-survey.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Today we are pleased to announce the [Rust contributor survey][survey]. This
+short, anonymous survey should only take a few minutes to fill out, but it will
+really help us in understanding what kinds of problems people experience when
+contributing to Rust so that we can try to address them. We value many
+different kinds of contributions, such as reporting problems, triaging issues,
+organizing meetups, etc, but this survey's purpose is to identify and eliminate
+barriers to entry for code contributions.
+
+The survey is targeting current, past, and future contributors to Rust. We're
+particularly interested in hearing from *new or prospective contributors*.
```
Comment (nikomatsakis) [inline posts/inside-rust/2020-05-27-contributor-survey.md:23]: Maybe we should include the text that clarifies that -- in this survey -- we're talking about *code contributions*?

Comment (mark-i-m) [reply posts/inside-rust/2020-05-27-contributor-survey.md]: I updated it to say "hearing from new or prospective code contributors", and we have some discussion in the previous para. Is there some other place I should clarify?

**Comment by nikomatsakis** [comment]
I'm satisfied, thanks @mark-i-m!

**Comment by mark-i-m** [comment]
Should we merge? The date in the survey is 2 weeks from today.

**Pull request merged**